### PR TITLE
delete variable

### PR DESCRIPTION
**Pull request by taminif**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description
I found variable don't used and re-declarate. So it deleted

### CODE REVIEW

File: redash/query_runner/mongodb.py [hunk 1]
```diff
@@ -251,7 +251,6 @@ def run_query(self, query, user):
                     step["$sort"] = SON(sort_list)
 
         if not aggregate:
-            s = None
```
Comment (arikfr) [inline redash/query_runner/mongodb.py]: It looks like this whole if block can be removed, as it being overridden below.

Comment (taminif) [reply redash/query_runner/mongodb.py]: that's exactly. thanks!

**Comment by taminif** [comment]
Thanks for the reviews @arikfr . I updated code. Please review once again!

**Comment by arikfr** [comment]
So it took me a while to get back to this, but it's merged now. Thanks.

**Pull request merged**